src/Shared/branding.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+#define AUTOMATION_VOICE_ID "BocoupAutomationVoice"
+#define AUTOMATION_VOICE_NAME "Bocoup Automation Voice"
+#define AUTOMATION_VOICE_VENDOR "Bocoup"
+#define AUTOMATION_VOICE_HOME "C:\\Program Files\\Bocoup Automation Voice"
```

Comment:
Should there be a newline at the end of this file?

Reply:
This could be `#define AUTOMATION_VOICE_HOME "C:\\Program Files\\" AUTOMATION_VOICE_NAME`. What do you think?

Reply:
I think it's more important to have the full path in the source code to facilitate text search by future contributors. Being DRY here means folks won't get an answer to the question, "Where is `Program Files\\Bocoup Automation Voice` defined?"